internal/pkg/git/git_test.go

```diff
@@ -1,5 +1,5 @@
 // nolint: forbidigo
-package git
+package git_test
```

Comment:
Musel jsem změnit kvůli circular dependency:
![CleanShot 2022-03-21 at 12 42 20](https://user-images.githubusercontent.com/215660/159254376-98ff155c-4eb6-400a-b720-b0ecdedce39f.jpg)

Reply:
ok, mohol si este importnut `git` ako `.`, tj, bez prefixu, ale to je jedno, moze to byt aj takto.

Reply:
hod ma potom znova na review ;) ked to bude uz cele ready